website/docs/docs/cloud/git/import-a-project-by-git-url.md

```diff
@@ -3,100 +3,117 @@ title: "Import a project by git URL"
 id: "import-a-project-by-git-url"
 ---
 
-In dbt Cloud, you can import a git repository from any valid git URL that points to a dbt project. There are a couple of important considerations to keep in mind when doing this:
+In dbt Cloud, you can import a git repository from any valid git URL that points to a dbt project. There are some important considerations to keep in mind when doing this.
 
 ## Git protocols
 You must use the `git@...` or `ssh:..`. version of your git URL, not the `https://...` version. dbt Cloud uses the SSH protocol to clone repositories, so dbt Cloud will be unable to clone repos supplied with the HTTP protocol.
 
+
 ## Managing Deploy Keys
 After importing a project by Git URL, dbt Cloud will generate a Deploy Key for your repository. You must provide this Deploy Key in the Repository configuration of your Git host. This Deploy Key should be configured to allow *read and write access* to the specified repositories.
 
 **Note**: Each dbt Cloud project will generate a different deploy key when connected to a repo, even if two projects are connected to the same repo. Both deploy keys will need to be supplied to your git provider.
 
-### GitHub
+## GitHub
 
 :::info Use GitHub?
 
 If you use GitHub, you can import your repo directly using [dbt Cloud's GitHub Application](/docs/cloud/git/connect-github). Connecting your repo via the GitHub Application [enables Continuous Integration](/docs/deploy/cloud-ci-job).
 
 :::
 
-To add a deploy key to a GitHub account, navigate to the Deploy keys tab of the settings page in your GitHub repository. After supplying a name for the deploy key and pasting in your deploy key (generated by dbt Cloud), be sure to check the "Allow write access" checkbox. After adding this key, dbt Cloud will be able to read and write files in your dbt project.
-
-See also:  [Adding a deploy key in GitHub](https://github.blog/2015-06-16-read-only-deploy-keys/)
+1. To add a deploy key to a GitHub account, navigate to the Deploy keys tab of the settings page in your GitHub repository. 
+2. After supplying a name for the deploy key and pasting in your deploy key (generated by dbt Cloud), be sure to check the **Allow write access** checkbox. 
+3. After adding this key, dbt Cloud will be able to read and write files in your dbt project.
```

Comment:
These don't all read like steps that are moving in order.  This might benefit from a broader rewrite if we want to list steps to a goal.

Reply:
ah ok, i haven't focused on the rewrite for this page. before adding this as an issue, do you think bullets would work?